src/sphinxcontrib/towncrier/_towncrier.py

```diff
@@ -22,9 +22,13 @@ def get_towncrier_config(
         final_config_path: Path,
 ) -> Dict[str, Any]:  # FIXME: add a better type  # pylint: disable=fixme
     """Return the towncrier config dictionary."""
-    try:
+    try:  # noqa: WPS229
         # Towncrier >= 19.9.0
-        return load_config_from_file(str(project_path), str(final_config_path))
+        config = load_config_from_file(str(project_path), str(final_config_path))
+        if dataclasses.is_dataclass(config):
```

Comment:
This could be put into the else-block.

Reply:
try... else? WPS raises WPS503 Found useless returning `else` statemen

Reply:
This means that it's possible to put this after the try-except block, saving one level of indentation.